File: svc-position/src/main/java/com/beancounter/position/model/ValuationData.java
package com.beancounter.position.model;

import com.beancounter.common.contracts.FxResponse;
import com.beancounter.common.contracts.PriceResponse;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class ValuationData {
  PriceResponse priceResponse;
  FxResponse fxResponse;
}

File: svc-position/src/main/kotlin/com/beancounter/position/model/ValuationData.kt
package com.beancounter.position.model

import com.beancounter.common.contracts.FxResponse
import com.beancounter.common.contracts.PriceResponse

/**
 * Rates and Prices necessary to values all positions.
 */
data class ValuationData(
    val priceResponse: PriceResponse = PriceResponse(),
    val fxResponse: FxResponse = FxResponse(),
)
